homeassistant/components/startca/sensor.py

```diff
@@ -143,7 +143,7 @@ def bytes_to_gb(value):
     async def async_update(self):
         """Get the Start.ca bandwidth data from the web service."""
         _LOGGER.debug("Updating Start.ca usage data")
-        url = f"https://www.start.ca/support/usage/api?key={self.api_key}"
+        url = f"https://www.start.ca/account/usage/api?key={self.api_key}"
```

Comment:
This integration actually needs some love, as all API specific code has to be part of a third-party library hosted on PyPi. Home Assistant should only interact with objects and not make direct calls to the API.

Nevertheless, this is a really small changes that fixes a bug, so I think it is fine to accept this. Just make sure the tests pass.

If you are interested in starting refactoring this integration, that would be highly appreciated. 👍

Reply:
Maybe in the future. I'd have to get comfortable enough with Python first.

Reply:
For this PR, the tests have to pass thought 😉